tests/unit/src/test/scala/tests/ScalaCliSuite.scala

```diff
@@ -1,5 +1,51 @@
 package tests
 
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.DebugUnresolvedMainClassParams
+import scala.meta.internal.metals.debug.TestDebugger
 import scala.meta.internal.metals.{BuildInfo => V}
 
-class ScalaCliSuite extends tests.BaseScalaCliSuite(V.scala213)
+class ScalaCliSuite extends tests.BaseScalaCliSuite(V.scala213) {
+
+  import scala.meta.internal.metals.JsonParser._
+
+  def startDebugging(
+      main: String,
+      buildTarget: String,
+  ): Future[TestDebugger] = {
+    server.startDebuggingUnresolved(
+      new DebugUnresolvedMainClassParams(main, buildTarget).toJson
+    )
+  }
+
+  test("base-native-run") {
+    cleanWorkspace()
+    for {
+      _ <- scalaCliInitialize(useBsp = true)(
+        s"""/MyMain.scala
+           |//> using scala "$scalaVersion"
+           |//> using platform "native"
```

Comment:
@lwronski according to https://scala-cli.virtuslab.org/docs/reference/directives/#platform `native` is incorrect value, correct one is `scala-native`. `scala-cli run --help` also yields `--platform scala-js|scala-native|jvm Specify platform`. However `native` is also accepted

Reply:
@WojciechMazur I'd like to use some scala-native specific in this case, just to be sure that `using platform "native"` is respected and indeed SN is used to run this snippet. Can you think of something which won't compile on JVM?

Reply:
Anything that uses 'scala.scalanative' package. So the simplest one would be 'import scala.scalanative._'

Reply:
Done, both for native and js.